Sources/YBottomSheet/Animation/BottomSheetDismissAnimator.swift

```diff
@@ -37,7 +37,9 @@ class BottomSheetDismissAnimator: BottomSheetAnimator {
             delay: .zero,
             options: [.beginFromCurrentState, sheet.appearance.dismissAnimationCurve]
         ) {
-            sheet.sheetView.frame = sheetFrame
+            if !self.isReduceMotionEnabled {
+                sheet.sheetView.frame = sheetFrame
+            }
```

Comment:
where's the else to fade out the sheet?
Although it should be if isReduceMotionEnabled.
Also the `self.` is redundant.